src/ign_src_TEST.cc

```diff
@@ -344,13 +344,12 @@ TEST(CmdLine, WorldListCustomServerPrettyOwner)
   std::stringstream stdErrBuffer;
   redirectIO(stdOutBuffer, stdErrBuffer);
 
-  EXPECT_TRUE(listWorlds("https://staging-fuel.ignitionrobotics.org", "nate"));
+  EXPECT_TRUE(listWorlds("https://staging-fuel.ignitionrobotics.org",
+      "OpenRobotics"));
 
   EXPECT_NE(stdOutBuffer.str().find(
         "https://staging-fuel.ignitionrobotics.org"), std::string::npos)
     << stdOutBuffer.str();
-  EXPECT_NE(stdOutBuffer.str().find("1 owners"), std::string::npos)
-      << stdOutBuffer.str();
```

Comment:
This expectation doesn't appear to be replaced with another one. Was it spurious?

Reply:
I removed that on purpose, because it's hard to assume an exact number of users for a given server. Right now for example there are none.